environments/xfel_sase1/__init__.py

```diff
@@ -0,0 +1,306 @@
+import time
+import numpy as np
+from badger import environment
+from badger.interface import Interface
+
+
+class Environment(environment.Environment):
+
+    name = 'xfel'
```

Comment:
Better keeping the name identical to the plugin folder name -- when listing plugins, Badger actually reads the folder names and shows them to the user, so it would be nice to use the same name for the plugin folder, the class field, and the name property in the `configs.yaml`, just to avoid confusion.

On second thought, it feels duplicated and confusing to have 3 names for one plugin... maybe we should only keep one...